content/blog/eng/functional-testing-symfony-guzzle.md

```diff
@@ -0,0 +1,319 @@
+---
+authors: ["carlo cappai"]
+comments: true
+date: "2021-04-06"
+draft: true
+share: true
+categories: [English, PHP, Testing, Symfony, Guzzle]
+title: "How to write a functional test with Symfony and Guzzle's mock handler"
+languageCode: "en-EN"
+type: "post"
+twitterImage: ''
+toc: true
+---
+
+# Introduction
+
+When we are writing a client to integrate an API in our systems it is important to test it to be sure you can handle every possible response.
```

Comment:
> "To be sure YOU can" 

Uniform the speaking person: you used "we" in the first part of the sentence, use "we" also here.